### PR TITLE
add kubesphere icon animation and katacoda scenario

### PR DESCRIPTION
**Pull request by FeynmanZhou**
Signed-off-by: FeynmanZhou <feynman@kubesphere.io>

### What type of PR is this?

/kind documentation

You can preview the updates [here](https://github.com/FeynmanZhou/kubesphere#readme).

### Which issue(s) this PR fixes:

This PR is an enhancement for README.

### Special notes for reviewers:
```
None.
```

### Does this PR introduced a user-facing change?

```release-note
Add kubesphere icon animation and katacoda scenario
```

### Additional documentation, usage docs, etc.:

```docs
None.
```

### CODE REVIEW

**Review by LinuxSuRen — COMMENTED** [review]
/lgtm

**Comment by LinuxSuRen** [comment]
Thanks for your contribution. But please review the PR template again.

image

**Comment by FeynmanZhou** [comment]
> Thanks for your contribution. But please review the PR template again.
> 
> image

Yes, I filled out the form above.

File: README.md [hunk 1]
```diff
@@ -160,7 +163,7 @@ We :heart: your contribution. The [community](https://github.com/kubesphere/comm
 
 Please submit any KubeSphere bugs, issues, and feature requests to [KubeSphere GitHub Issue](https://github.com/kubesphere/kubesphere/issues).
 
-## Who are using KubeSphere
+## Who are Using KubeSphere
```
Comment (zryfish) [inline README.md:166]: Why do we capitalize `using`?

Comment (FeynmanZhou) [reply README.md]: I updated all headings to the sentence case according to the [Google Doc Style Guide](https://developers.google.com/style/capitalization).

**Pull request merged**